R/tbl_survfit_quantiles.R

```diff
@@ -0,0 +1,260 @@
+#' Survival Quantiles
+#'
+#' Create a gtsummary table with Kaplan-Meier estimated survival quantiles.
+#' If you must further customize the way these results are presented,
+#' see the Details section below for the full details.
+#'
+#' @param data (`data.frame`)\cr
+#'   A data frame
+#' @param y (`string` of `expression`)\cr
+#'   A string or expression with the survival outcome, e.g. `survival::Surv(time, status)`.
+#'   The default value is
+#'   `survival::Surv(time = AVAL, event = 1 - CNSR, type = "right", origin = 0)`.
```

Comment:
maybe I would add see `ggsurvfit::Surv_CNSR` for further details? It is fine without too as it is not used directly

Reply:
I think we don't need to reference it because it does potentially raise an issue when using with model.frame() and `all.vars()` because it can't read the variable names